### PR TITLE
vc: Enable the testsuite, excluding tests failing on Zen2

### PR DESCRIPTION
**Pull request by bernhardkaindl**
This fixes running the testsuite, it adds the package virtest for it.

### CODE REVIEW

**Review by iarspider — APPROVED** [review]
LGTM. A bit unsure about changing `CPATH` (spack always complains when it is modified), will let others comment

**Comment by iarspider** [comment]
@tldahlgren I'm not authorized to merge this (fortunately).

File: var/spack/repos/builtin/packages/vc/package.py [hunk 1]
```diff
@@ -23,6 +23,15 @@ class Vc(CMakePackage):
             values=('Debug', 'Release', 'RelWithDebug',
                     'RelWithDebInfo', 'MinSizeRel'))
 
+    # Build fails without it when --test is used, can't be type='test':
+    depends_on('virtest')
+
+    def patch(self):
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/vc/package.py:29]: Do the dependency and patch always apply or only when the target is `zen2`?

Comment (bernhardkaindl) [reply var/spack/repos/builtin/packages/vc/package.py:29]: @tldahlgren No, the build is not affected by the architecture in any way.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**